db/migrate/20200907114036_create_computacenter_user_ledger.rb

```diff
@@ -0,0 +1,25 @@
+class CreateComputacenterUserLedger < ActiveRecord::Migration[6.0]
+  def change
+    create_table :computacenter_user_ledgers do |t|
+      t.integer :user_id
+      t.text :first_name
+      t.text :last_name
+      t.text :email
```

Comment:
Can we please change this for consistency?

```suggestion
      t.text :email_address
```